frontend/src/css/modules/sidebar.scss

```diff
@@ -1,12 +1,13 @@
 .ev-sidebar {
     display: block;
-    height: 100vh;
+    height: calc(100vh - 120px);
```

Comment:
No we can't use percentage here, as sidebar should of fix height and there is no need to make it responsive height wise. So sometimes it's good to use fix value.